exdb/tests/integration_tests.py

```diff
@@ -31,14 +34,28 @@ def create_keyword(self, name="Test Keyword"):
     def create_experience(self, exp_status, attendance=0):
         """Creates and returns an experience object with status,
         start_time, end_time and/or name of your choice"""
-        return Experience.objects.get_or_create(author=self.test_user, name="Test Experience", description="test description", start_datetime=self.test_date,
-                                                end_datetime=(self.test_date + timedelta(days=1)), type=self.create_type(), sub_type=self.create_sub_type(), goal="Test Goal", audience="b",
-                                                status=exp_status, attendance=attendance)[0]
+        return Experience.objects.get_or_create(
+            author=self.clients['ra'].user_object,
+            name="Test Experience",
+            description="test description",
+            start_datetime=self.test_date,
+            end_datetime=(self.test_date + timedelta(days=1)),
+            type=self.create_type(),
+            sub_type=self.create_sub_type(),
+            goal="Test Goal",
+            audience="b",
+            status=exp_status,
+            attendance=attendance
+        )[0]
 
     def create_experience_comment(self, exp, message="Test message"):
         """Creates experience comment, must pass an experience"""
         return ExperienceComment.objects.get_or_create(
-            experience=exp, message=message, author=self.test_user, timestamp=self.test_date)[0]
+            experience=exp,
+            message=message,
+            author=self.clients['ra'].user_object,
```

Comment:
If we want to make this as realistic as possible, we should probably have the Hall-Staff be the author on the experience comment as (at least right now) they are the only ones allowed to add a comment.